src/main/java/edu/harvard/iq/dataverse/branding/BrandingUtil.java

```diff
@@ -1,22 +1,39 @@
 package edu.harvard.iq.dataverse.branding;
 
+import edu.harvard.iq.dataverse.DataverseServiceBean;
 import edu.harvard.iq.dataverse.util.BundleUtil;
 import java.util.Arrays;
+import javax.ejb.EJB;
+import javax.ejb.Singleton;
 import javax.mail.internet.InternetAddress;
 
+import org.eclipse.microprofile.config.ConfigProvider;
+
+@Singleton
 public class BrandingUtil {
 
-    public static String getInstallationBrandName(String rootDataverseName) {
-        return rootDataverseName;
+    @EJB
+    static DataverseServiceBean dataverseService;
+
+    public static String getInstallationBrandName() {
+        return ConfigProvider.getConfig().getOptionalValue("dataverse.branding.installation.name", String.class)
```

Comment:
While I'm still baffled that injecting stuff into a `static` member is possible with EJB (I don't think it is with CDI), do you actually want to use the programmatic call or should we go for the injected version?

Reply:
I was unable to get the injection of the variable to work, so I picked this way. That said, I've now had to remove this one for other reasons. It's possible that @Injecting could work for the exclude setting used in the DDIExportUtil class - I didn't try there.